src/dbt/kipptaf/models/reporting/staging/stg_reporting__src_reporting__gradebook_flags.sql

```diff
@@ -0,0 +1,25 @@
+with
+
+    source as (
+
+        select * from {{ source("reporting", "src_reporting__gradebook_flags") }}
```

Comment:
this line should be the entirety of the view, unless you plan on doing something else with it

Reply:
also address the linter error